src/actions/__tests__/cursorDown.ts

```diff
@@ -88,6 +89,28 @@ describe('normal view', () => {
     const stateNew = cursorDown(store.getState())
     expectPathToEqual(stateNew, stateNew.cursor, ['a', 'm'])
   })
+
+  it('move cursor from empty thought to next thought', () => {
```

Comment:
We can make this test description a little more specific. See 45d582df01.

Reply:
I got it.